recipes/restraintlib/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "2021.7.7" %}
+{% set sha256 = "e3d07556034aa0bfffcff55c6051aa55099c8196a2c4ab77450f9a872863d376" %}
+
+package:
+  name: restraintlib
+  version: {{ version }}
+
+source:
+  url: https://github.com/mkowiel/restraintlib/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+  host:
+    - pip
+    - python
```

Can just merge these into a `host` section. Also no need for `pytest` since you list that for running the tests:
```suggestion
 host:
    - pip
    - setuptools
    - python
```